### PR TITLE
Fixed page reload 

### PR DESCRIPTION
**Pull request by jasehackman**
git checkout -b jh-page_reload_fix
git pull origin jh-page_reload_fix

Please pull down and test all buttons and searches. I made a lot of changes that change how our app functions

I added a few functions to replace the window reload command we were using. 

- clear out our buildElement variable before each use

- clear out div that holds results before and after each use

- update itinerary by calling a fetch function

### CODE REVIEW

**Comment by wesseln1** [comment]
Everything seemed to work great for me! 👍 :shipit: 🦑

File: src/scripts/food/food_filterdata.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const foodFilter = (foodObj) => {
 
 // Loop through my array and collect the name and location of each restaurant. T
 const foodArrayLoop = (newArray) => {
+  buildClear();
```
Comment (CurtainUp) [inline src/scripts/food/food_filterdata.js]: Will we all need to add this to our filter.js?  Or can this be added as the first function in our search button event listeners? Just wondering if there's a place to make this DRY-er

Comment (CurtainUp) [reply src/scripts/food/food_filterdata.js]: ... I see now that you DID add it to our files. Haaaaaaa nevermind.

**Comment by CurtainUp** [comment]
🦆👍

**Comment by robbyhecht** [comment]
👍 works for me

**Pull request merged**